src/aspnetcore/eng/targets/CSharp.Common.targets

```diff
@@ -94,7 +94,12 @@
         <_Parameter2>$(RepositoryUrl)/tree/$(SourceRevisionId)</_Parameter2>
       </AssemblyAttribute>
 
-      <AssemblyAttribute Include="System.Reflection.AssemblyMetadataAttribute" Condition="'$(Serviceable)' == 'true'">
+      <AssemblyAttribute Include="System.Reflection.AssemblyMetadataAttribute" Condition="'$(RepoOriginalSourceRevisionId)' != ''">
+        <_Parameter1>OriginalRepoCommitHash</_Parameter1>
```

Comment:
nit: maybe call it `AspNetCoreRepoCommitHash` so it's a bit clearer instead of "original" ?